roles/network_plugin/calico/tasks/install.yml

```diff
@@ -223,7 +223,7 @@
               "name": "{{ calico_pool_name }}",
             },
             "spec": {
-              "blockSize": {{ calico_pool_blocksize | default(kube_network_node_prefix) }},
+              "blockSize": {{ calico_pool_blocksize }},
```

Comment:
default(26) ?

Reply:
That's already in defaults/main.yml, I'd rather avoid duplicating.